### PR TITLE
[TE] frontend - baseline event tweaks

### PR DESCRIPTION
**Pull request by apucher**
* move logic for tagging events as "baseline" from backend to frontend

* display baseline event in light color

* tweak/fix labels and formatting of events

### CODE REVIEW

File: thirdeye/thirdeye-frontend/app/actions/events.js [hunk 1]
```diff
@@ -87,26 +91,28 @@ const assignEventTimeInfo = (event, anomalyStart, anomalyEnd, baselineStart, bas
...
     relEnd = event.end - anomalyStart;
     relDuration = event.end - event.start;
   }
 
-  var isBaseline = false;
-  var displayStart = event.start;
-  var displayEnd = event.end;
-  if (baselineStart <= displayStart && displayStart < baselineEnd) {
+  let isBaseline = false;
+  let displayStart = event.start;
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/actions/events.js]: fyi, you can use es6's destructuring to do this: 
```
let { 
  start : displayStart, 
  end: displayEnd,
  label: displayLabel
} = event; 
``` 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring

File: thirdeye/thirdeye-frontend/app/pods/components/anomaly-graph/component.js [hunk 2]
```diff
@@ -9,7 +9,14 @@ const COLOR_MAPPING = {
   purple: '#9896F2',
   red: '#FF6C70',
   green: '#6BAF49',
-  pink: '#FF61b6'
+  pink: '#FF61b6',
+  light_blue: '#73EAFF',
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/pods/components/anomaly-graph/component.js]: are those colors from go/patterns ?

File: thirdeye/thirdeye-frontend/app/styles/components/anomaly-graph.scss [hunk 3]
```diff
@@ -127,12 +127,19 @@
 
 $colors:
   "blue" $te-blue,
+  "light_blue" $te-orange,
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/styles/components/anomaly-graph.scss]: is this mapping accurate?

Comment (apucher) [reply thirdeye/thirdeye-frontend/app/styles/components/anomaly-graph.scss]: It's very much a hack. I want the dots to be displayed in bright color, but the description in readable, regular color. One alternative would be to hard-code a "lighten" in the anomaly graph wrapper when it encounters a baseline event (event.isBaseline == true). thoughts?

File: thirdeye/thirdeye-frontend/app/pods/components/anomaly-graph/component.js (hunk 2 above)
Comment (apucher) [reply thirdeye/thirdeye-frontend/app/pods/components/anomaly-graph/component.js]: absolutely not. I'll review the docs

File: thirdeye/thirdeye-frontend/app/styles/components/anomaly-graph.scss (hunk 3 above)
Comment (justYves) [reply thirdeye/thirdeye-frontend/app/styles/components/anomaly-graph.scss]: SASS provides a lighten mixin already. 
doc: http://sass-lang.com/documentation/Sass/Script/Functions.html#lighten-instance_method
usage: https://codepen.io/baudoin/pen/HdliD

As always, come find me if you have any questions!

**Pull request merged**